.mergify.yml

```diff
@@ -58,3 +58,12 @@ pull_request_rules:
       review:
         type: APPROVE
         message: A change to `Changelog.md`? I can handle that!
+  - name: Auto-approve dependabot PR
+    conditions:
+      - files=doc/docusaurus/package-lock.json
+      - "#files=1"
+      - label=automerge-squash
```

Comment:
Not sure if these PRs get that label. I'd assume not, but you can use mergify to label them.

Reply:
Oh, I thought that entry is putting a label on it. Looking further...

Reply:
b28f41a34